website/cue/reference/components/sinks/aws_cloudwatch_logs.cue

```diff
@@ -72,35 +73,8 @@ components: sinks: aws_cloudwatch_logs: components._aws & {
 		warnings: []
 	}
 
-	configuration: {
-		create_missing_group: {
-			common:      true
-			description: "Dynamically create a [log group](\(urls.aws_cloudwatch_logs_group_name)) if it does not already exist. This will ignore `create_missing_stream` directly after creating the group and will create the first stream."
-			required:    false
-			type: bool: default: true
-		}
-		create_missing_stream: {
-			common:      true
-			description: "Dynamically create a [log stream](\(urls.aws_cloudwatch_logs_stream_name)) if it does not already exist."
-			required:    false
-			type: bool: default: true
-		}
-		group_name: {
-			description: "The [group name](\(urls.aws_cloudwatch_logs_group_name)) of the target CloudWatch Logs stream."
-			required:    true
-			type: string: {
-				examples: ["group-name", "{{ file }}"]
-				syntax: "template"
-			}
-		}
-		stream_name: {
-			description: "The [stream name](\(urls.aws_cloudwatch_logs_stream_name)) of the target CloudWatch Logs stream. Note that there can only be one writer to a log stream at a time so if you are running multiple vectors all writing to the same log group, include a identifier in the stream name that is guaranteed to be unique by vector instance (for example, you might choose `host`)"
-			required:    true
-			type: string: {
-				examples: ["{{ host }}", "%Y-%m-%d", "stream-name"]
-				syntax: "template"
-			}
-		}
+	configuration: base.components.sinks.aws_cloudwatch_logs.configuration & {
+		_aws_include: false
```

Comment:
What is this `_aws_include`?

Reply:
I added it when working on the sources, [here](https://github.com/vectordotdev/vector/blob/master/website/cue/reference/components/aws.cue#L7). It's similar to the `auto_generated` option that prevents fields from being included by both the original cue and the new "base" cue.

Reply:
Presumably that whole thing can be deleted once all the sinks are moved over?

Reply:
I _think_ so!